src/lib/icons/IconLockOpen.svelte

```diff
@@ -8,18 +8,39 @@
 <svg
   width={size}
   height={size}
-  viewBox="0 0 20 20"
+  viewBox="0 0 21 21"
```

Comment:
If I copy SVG from Figma, I get:
```
<svg width="21" height="21" viewBox="0 0 21 21" fill="none" xmlns="http://www.w3.org/2000/svg">
<path d="M6.70651 8.00232H15.0373C16.0032 8.00232 16.7865 8.78561 16.7865 9.75232V15.5857C16.7865 16.5524 16.0032 17.3357 15.0373 17.3357H6.70651C5.74065 17.3357 4.95728 16.5524 4.95728 15.5857V9.75232C4.95728 8.78561 5.74065 8.00232 6.70651 8.00232Z" stroke="white" stroke-width="1.5"/>
<path d="M13.6492 7.25232V4.83899C13.6492 3.39585 14.8191 2.22595 16.2622 2.22595V2.22595C17.7053 2.22595 18.8752 3.39585 18.8752 4.83899V5.09517" stroke="white" stroke-width="1.5" stroke-linecap="round"/>
<path d="M10.8699 12.6691H10.8749" stroke="white" stroke-width="2" stroke-linecap="round" stroke-linejoin="round"/>
</svg>
```
This has both a size and viewBox of 21.
But then if we use it with our default of size 20, but keep the viewBox of 21, then it ends up being smaller than intended, right?

I'm checking this with Artem as well.

Reply:
Correct, because of the clip-path is 20x20 px.

Reply:
The icons were updated, so there is no more viewBox 21x21.